integration_tests/suite/test_queue_stats.py

```diff
@@ -323,3 +330,125 @@ def test_list_queue_statistics_multiple_queues_same_datetime(self):
 
     def test_get_queue_no_interval_returns_from_to_until(self):
         pass
+
+    # fmt: off
+    @stat_queue_periodic({'queue_id': 1, 'time': '2020-10-06 4:00:00', 'total': 1, 'answered': 1})
+    @stat_queue_periodic({'queue_id': 1, 'time': '2020-10-06 5:00:00', 'total': 36, 'closed': 1, 'abandoned': 2, 'joinempty': 3, 'leaveempty': 4, 'timeout': 5, 'divert_ca_ratio': 6, 'divert_waittime': 7, 'full': 8})
+    @stat_queue_periodic({'queue_id': 1, 'time': '2020-10-06 23:00:00', 'total': 1, 'answered': 1})
+    @stat_queue_periodic({'queue_id': 1, 'time': '2020-10-07 00:00:00', 'total': 1, 'answered': 1})
+    # fmt: on
+    def test_get_queue_interval_by_hour(self):
+        results = self.call_logd.queue_statistics.get_by_id(
+            queue_id=1,
+            from_='2020-10-06 00:00:00', until='2020-10-07 00:00:00',
+            interval='hour',
+        )
+        assert_that(results, has_entries(total=equal_to(25)))
+
+        assert_that(
+            results['items'],
+            has_item(has_entries({
```

Comment:
has_entries can also be used like this to reduce the amount of typing
```python
has_entries(
    from='2020-10-06T04:00:00+00:00',
    until='2020-10-06T05:00:00+00:00'
   ...
)
```
This does not have to be changed, it's just my personal preference that I'm trying to spread ;)
And it does not work if the key contains an `-`

Reply:
I tried it, but `from` is a reserved word :frowning_face: I have to do `**{'from': '...'}` to use it as a keyword